### PR TITLE
Migrates CRA configuration to Vite

### PR DESCRIPTION
**Pull request by Genyus**
Updates all exercises to use Vite instead of CRA

[TECH-557](https://codingblackfemales.atlassian.net/browse/TECH-557)

[TECH-557]: https://codingblackfemales.atlassian.net/browse/TECH-557?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: first-react-app/README.md [hunk 1]
```diff
@@ -7,46 +7,50 @@ pwd
...
 ```
 
-## NPM and Node.js Versions
+## Prerequisites
 
-To run the `create-react-app` you’ll need to have `Node >= 14.0.0` and `npm >= 5.6` on your machine. 
+To run `vite`, you'll need to have Node 18.x or 20.x installed on your machine. 
 
-To check your node.js version run the following command:
+To check your node.jsx version run the following command:
```
Comment (efuaakum) [inline first-react-app/README.md]: Minor typo: node.jsx should be node.js

**Comment by efuaakum** [comment]
@Genyus To summarise the changes, is it correct to say that for each CRA project/task the following changes have been made?:
- Add dependencies for Vite
- Update start, build commands for Vite
- Rename all .js files to .jsx 
- Update README.md instructions on how to run

Have I missed anything off the list?

**Comment by Genyus** [comment]
@efuaakum Pretty much. The only other change has been migrating any tests from Jest to Vitest

**Comment by tanyapowell** [comment]
What's the difference between Vitest and Jest?

**Comment by Genyus** [comment]
Vitest is largely API-compatible with Jest (for everyday scenarios, at least), but more performant and easier to implement in Vite-based projects

**Review by efuaakum — APPROVED** [review]

**Pull request merged**